int32[] indices_considered           # indices of grasps seen by the user
int32 index_selected                 # index of grasp selected for execution